core/array/any_spec.rb

```diff
@@ -0,0 +1,30 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe "Array#any?" do
+  describe 'with no block given' do 
+    it "checks if there are any members of an array" do 
```

Comment:
Sure. Although, I am having trouble constructing a hash with falsy values such that `any?` is false.

`{nil => nil}.any? #=> true`
`{false => false}.any? #=> true`
`{nil => false}.any? #=> true`